zilliqa/src/api/zil.rs

```diff
@@ -470,3 +481,144 @@ fn get_smart_contracts(params: Params, node: &Arc<Mutex<Node>>) -> Result<Vec<Sm
 
     Ok(contracts)
 }
+
+fn get_example_ds_block_verbose(dsblocknum: u64, txblocknum: u64) -> DSBlockVerbose {
+    DSBlockVerbose {
+        B1: vec![false, false, false],
+        B2: vec![false, false],
+        CS1: String::from("FBA696961142862169D03EED67DD302EAB91333CBC4EEFE7EDB230515DA31DC1B9746EEEE5E7C105685E22C483B1021867B3775D30215CA66D5D81543E9FE8B5"),
+        PrevDSHash: String::from("585373fb2c607b324afbe8f592e43b40d0091bbcef56c158e0879ced69648c8e"),
+        header: DSBlockHeaderVerbose {
+            BlockNum: dsblocknum.to_string(),
+            CommitteeHash: String::from("da38b3b21b26b71835bb1545246a0a248f97003de302ae20d70aeaf854403029"),
+            Difficulty: 95,
+            DifficultyDS: 156,
+            EpochNum: txblocknum.to_string(),
+            GasPrice: String::from("2000000000"),
+            MembersEjected: vec![
```

Comment:
I think we should leave most of these lists blank if we don't have valid data to put in them, rather than making stuff up. I'd rather a client breaks because (e.g.) `PoWWinners` is empty in ZQ2 (since we don't have any proof of work) rather than continuing to work but with nonsense data.

Reply:
Will fix in follow up PR